presto-docs/src/main/sphinx/release/release-339.rst

```diff
@@ -0,0 +1,70 @@
+Release 339 (XX Jul 2020)
+=========================
+
+General Changes
+---------------
+
+* Add :func:`approx_most_frequent`. (:issue:`3425`)
+* Physical bytes scan limit for queries can be configured via ``query.max-scan-physical-bytes`` configuration property
+  and ``query_max_scan_physical_bytes`` session property. (:issue:`4075`)
+* Remove support for addition and subtraction between ``TIME`` and ``INTERVAL YEAR TO MONTH`` types. (:issue:`4308`)
+* Fix planning failure when join criteria contains subqueries. (:issue:`4380`)
+* Fix failure when subqueries appear in window function arguments. (:issue:`4127`)
+* Fix failure when subquery in WITH clause contains hidden columns. (:issue:`4423`)
+* Fix failure when referring to type names with different case ``GROUP BY`` queries. (:issue:`2960`)
+* Fix failure for queries involving ``DISTINCT`` when expressions in ``ORDER BY`` clause differ from expressions in ``SELECT`` clause by case. (:issue:`4233`)
+* Fix incorrect type reporting for ``TIMESTAMP`` and ``TIMESTAMP WITH TIME ZONE`` for legacy clients. (:issue:`4408`)
+* Fix failure when querying nested ``TIMESTAMP`` or ``TIMESTAMP WITH TIME ZONE`` for legacy clients. (:issue:`4475`, :issue:`4425`)
+
+JDBC Driver Changes
+-------------------
+
+* Fix reading ``timestamp`` and ``timestamp with time zone`` values with year number higher than 9999
+  or negative. (:issue:`4364`
+
+Druid Connector Changes
+-----------------------
+
+* Fix handling of table and column names containing non-ASCII characters. (:issue:`4312`)
+
+Hive Connector Changes
+----------------------
+
+* Make ``location`` parameter of the ``system.register_partition`` procedure optional. (:issue:`4443`)
+* Remove requirement to configure ``metastore.storage.schema.reader.impl`` in Hive 3.x metastore
+  to let Presto access CSV tables. (:issue:`4457`)
+* Fix a query failure when reading from Parquet file containing ``real`` or ``double`` ``NaN`` values,
+  if the file was written by certain non-conforming writer. (:issue:`4267`)
+
+Kafka Connector Changes
+-----------------------
+
+* Add insert support for Avro. (:issue:`4418`)
+* Add insert support for CSV. (:issue:`4287`)
+
+Kudu Connector Changes
+----------------------
+
+* Add support for grouped execution. It can be enabled with ``kudu.grouped-execution.enabled`` configuration property and ``grouped_execution`` session property. (:issue:`3715`)
+
+MongoDB Connector Changes
+-------------------------
+
+* Upgrade MongoDB driver to ``3.8.0`` from ``3.6.0`` to resolve query failure when using Azure Cosmos DB. (:issue:`4415`)
+
+Oracle Connector Changes
+------------------------
+
+* Fix propagating JDBC credentials if connection pooling is enabled. (:issue:`4430`)
+
+Phoenix Connector Changes
+-------------------------
+
+* Fix handling of row key definition with white space. (:issue:`3251`)
+
+SPI Changes
+-----------
+
+* Allow connectors to wait for dynamic filters before splits are generated. (:issue:`4224`)
```

Comment:
IMO this should include a kind of pointer, where to start if i want to utilize this in my connector.
e.g. i do not know where to look for this.

Reply:
Would something like:
```
(dynamic filter future can be obtained by calling ``DynamicFilter#isBlocked`` method).
```
be OK?

Reply:
Maybe

```
* Allow connectors to wait for dynamic filters before splits are generated. This can be
  achieved by consulting ``isBlocked`` method of the ``DynamicFilter`` objected
  passed to ``ConnectorSplitManager#getSplits`` method. (:issue:`4224`)
```

?